nvtabular/ops/base.py

```diff
@@ -100,10 +99,46 @@ def label(self) -> str:
 
     @property
     def supports(self) -> Supports:
-        """ Returns what kind of data representation this operator supports """
+        """Returns what kind of data representation this operator supports"""
         return Supports.CPU_DATAFRAME | Supports.GPU_DATAFRAME
 
-    def inference_initialize(self, columns: ColumnNames, model_config: dict) -> Optional[Operator]:
+    def inference_initialize(self, columns: Columns, model_config: dict) -> Optional[Operator]:
         """Configures this operator for use in inference. May return a different operator to use
         instead of the one configured for use during training"""
         return None
+
+
+class AddMetadata(Operator):
```

Comment:
This should not be in the base class file... Its a stand alone operator. should be in metadata.py in ops or something.